taxi/views.py

```diff
@@ -30,10 +31,30 @@ def index(request):
 class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     model = Manufacturer
     context_object_name = "manufacturer_list"
-    template_name = "taxi/manufacturer_list.html"
+    # template_name = "taxi/manufacturer_list.html"
```

Comment:
Code Efficiency: `template_name` is commented out. It's good that you don't override `template_name` if it's the same as the default one.